src/common/components/navigation/Menu.tsx

```diff
@@ -82,6 +116,22 @@ export interface MenuProps {
 
 export const Menu: React.FunctionComponent<MenuProps> = (props) => {
   const classes = useStyles();
+  const { url } = useRouteMatch();
+  const { pathname } = useLocation();
+
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const handleSetSelectedIndex = () => {
+    menuItems.findIndex((item, i) => {
```

Comment:
It still gives a legit warning:
```
./src/common/components/navigation/Menu.tsx
  Line 125:35:  Expected to return a value in arrow function  array-callback-return
```
The **result** of `findIndex()` should give the value for `setSelectedIndex()`.